tools-v2/pkg/cli/command/curvebs/status/mds/mds.go

```diff
@@ -59,8 +59,8 @@ func NewMdsCommand() *cobra.Command {
 }
 
 func (mCmd *MdsCommand) AddFlags() {
-	config.AddBsMdsFlagOption(mCmd.Cmd)
 	config.AddHttpTimeoutFlag(mCmd.Cmd)
```

Comment:
no need modify?

Reply:
> no need modify?

same question as forementioned, I think this order is more good-looking...

Reply:
done